UnitystationLauncher/ViewModels/InstallationsPanelViewModel.cs

```diff
@@ -92,10 +87,61 @@ private async Task OnAutoRemoveChangedAsync()
             }
         }
 
+        private void InitializeInstallationsList()
+        {
+            Log.Information("Initializing installations list...");
+            List<Installation> installations = _installationService.GetInstallations();
+
+            foreach (Installation installation in installations)
+            {
+                InstallationViews.Add(new(installation, _installationService));
+            }
+
+            Log.Information("Scheduling periodic refresh for installations list...");
+            RxApp.MainThreadScheduler.SchedulePeriodic(_refreshInterval, RefreshInstallationsList);
+        }
+
+        private void RefreshInstallationsList()
+        {
+            List<Installation> installations = _installationService.GetInstallations();
+
+            // Add new
+            foreach (Installation installation in installations)
+            {
+                InstallationViewModel? viewModel = InstallationViews.FirstOrDefault(view => view.Installation.InstallationId == installation.InstallationId);
+
+                if (viewModel == null)
+                {
+                    InstallationViews.Add(new(installation, _installationService));
+                }
+            }
+
+            // Remove old
+            for (int i = 0; i < InstallationViews.Count; i++)
+            {
+                InstallationViewModel viewModel = InstallationViews[i];
+                Installation? installation = installations.FirstOrDefault(inst => inst.InstallationId == viewModel.Installation.InstallationId);
+
+                if (installation == null)
+                {
+                    InstallationViews.Remove(viewModel);
+                    i--;
```

Comment:
wait, what is this doing?

Reply:
ah I see. Iterate from last to first instead

Reply:
This is just bumping the iterator back one when it removes the element its currently on so that whatever is next in the list doesn't get skipped.

Reply:
yeah, I understood that. Still, I think it is clearer to iterate backwards so the index doesn't change when you delete an element.

Reply:
Made it so the remove operation iterates backwards both here and in the servers panel